tuf/client/updater.py

```diff
@@ -2834,15 +2843,17 @@ def download_target(self, target, destination_directory):
     target_file_object = self._get_target_file(target_filepath, trusted_length,
                                                trusted_hashes)
    
-    # We acquired a target file object from a mirror.  Move the file into
-    # place (i.e., locally to 'destination_directory').  Note: join() discards
-    # 'destination_directory' if 'target_path' contains a leading path separator
-    # (i.e., is treated as an absolute path).
+    # We acquired a target file object from a mirror.  Move the file into place
+    # (i.e., locally to 'destination_directory').  Note: join() discards
+    # 'destination_directory' if 'target_path' contains a leading path
+    # separator (i.e., is treated as an absolute path).
     destination = os.path.join(destination_directory,
                                target_filepath.lstrip(os.sep))
     destination = os.path.abspath(destination)
     target_dirpath = os.path.dirname(destination)
-    
+   
+    # When attempting to create the root directory of 'target_dirpath', pass on
```

Comment:
I think you mean: "**Raise** on all OSError exceptions except if the **leaf** directory already exists." No?